docusaurus/docs/protocol/primitives/probabilistic_proofs.md

```diff
@@ -1,23 +1,514 @@
+
+Ideally, we would choose `2σ` above the Claim `μ` such that `97.3%` fall of all Claims require a Proof.
+
+##### Non-Normal Distribution
+
+In practice, rewards are not normally distributed, so we can choose an arbitrary value (e.g. `p95`)
+such that 95% of Claims fall into the category of requiring a proof.
+
+#### Considerations for `ProofRequestProbability` (`p`)
+
+:::note
+
+See [Pocket_Network_Morse_Probabilistic_Proofs.ipynb](./Pocket_Network_Morse_Probabilistic_Proofs.ipynb) for more details from Morse backing the fact that the majority of the block space is taken up by Proofs.
+
+:::
+
+Accept the fact that the majority of the block space is taken up by Proofs.
+
+The number of relays in the network scales inversely to `ProofRequestProbability`. For example:
+
+- `ProofRequestProbability` = 0.5 -> 2x scale
+- `ProofRequestProbability` = 0.25 -> 4x scale
+- `ProofRequestProbability` = 0.1 -> 10x scale
+- `ProofRequestProbability` = 0.01 -> 100x scale
+- `ProofRequestProbability` = 0.001 -> 1000x scale
+
+##### Maximizing `Pr(X<=k)` to ensure `k or less` failures (Supplier escapes without penalty)
+
+When selecting a value for `p`, our goal is not to maximize `Pr(X=k)`, but rather
+maximize `Pr(X<=k)` to ensure `k or less` failures (Supplier escapes without penalty).
+
+This does not affect the expected reward calculations above, but gives a different
+perspective of what the probabilities of success and failure are.
+
+## Conclusions for Modelling
+
+By modeling the attack using a geometric distributions and calculating expected values, we can:
+
+- Determine `R = ProofRequirementThreshold` using statical onchain data
+- Manually adjust `p = ProofRequestProbability` to adjust scalability
+- Compute `S ≤ SupplierMinStake` to deter dishonest behavior
+- Determine the necessary penalty `S` to deter dishonest behavior.
+- Ensure that honest Suppliers remain profitable while dishonest Suppliers face negative expected profits.
+
+This approach allows the network to scale by reducing the number of on-chain Proofs while maintaining economic incentives that discourage dishonest behavior.
+
+## Morse Based Value Selection
+
+As of writing (October 2024), Shannon is not live and only Morse can be used to approximate realistic values.
+
+### Selecting `ProofRequirementThreshold`
+
+Choose `R = 20` since it is greater than `p95` of all Claims collected in Morse. Units are in `POKT`.
+
+See the original proposal from Morse available in [probabilistic_proofs_morse.md](./probabilistic_proofs_morse.md)
+and [Pocket_Network_Morse_Probabilistic_Proofs.ipynb](./Pocket_Network_Morse_Probabilistic_Proofs.ipynb) for supporting data.
+
+$$ R = 20 $$
+
+### Calculating `p`: `ProofRequestProbability`
+
+Choose `p = 0.01` to ensure high scalability.
+
+$$ E[K] = \frac{q}{p} = \frac{0.99}{0.01} = 99 $$
+
+### Calculating `S`: `ProofMissingPenalty`
+
+$$ S = R \cdot E[K] = 20 \cdot 99 = 1980 ≈ 2,000 $$
+
+## Future Work
```

Comment:
We should investigate above threshold attacks possibilities.
1. Supplier adds fake serviced relays to the SMT.
2. Submits the claim.
3. If the closest proof corresponds to a:
  a. Fake relay ->  Do not submit the proof and face slashing.
  b. Legit relay -> Submit the proof and be rewarded for fake relays.

Reply:
This is interesting, you talk about inflating the tree with fake relays. I think that the effect will be a reduction of the catch probability that is proportional to the fake relays ratio.
Supose that you inflate your tree by a X%, then you have a a chance of being requested a proof given by `ProofRequestProbability` and a `X/100` chance that the requested proof is a fake relay.
I think that in practice this can be modeled as a reduced `ProofRequestProbability`, so we can add a security factor there.